### DISCUSSION TITLE
WHATS ACTUALLY NEXT

### DISCUSSION THREAD

**Discussion by mayo78** [mayo78/FNF-PsychEngine-BambiAddon#10] [Announcements]
so i actually haven't really taken that much of a look at newer versions of psych engine, my thoughts? uhhhh its kinda over complicated now, which is good for psych mods, but bad for standalone mods, which is what this addon was supposed to create. in my opinion doing stuff like note skins is really way easier if you just code a simple system compared to using menus and stuff. same with note type skins. hscript got a few additions but hscript is literally just access to the entirety of flixel and psych so its only just kinda nice. i think at this point if i were to update this thing and port everything over, it would honestly be easier to just use source code. honestly im not really sure if it would be good to use the current version of psych for modding since you'd have to really gut or move to the side of all the soft modded stuff, i think i might use dave engine for my next fnf mod if i make one since its since its mostly hard coded and simple.

tldr new psych too softmoded would be stupid to port everything over use source code instead if you want a standalone build
